sdcm/tester.py

```diff
@@ -326,6 +326,19 @@ def run(self, result=None):
         result = super(ClusterTester, self).run(self.result)
         return result
 
+    @property
+    def reliable_replication_factor(self) -> int:
+        """
+        Calculates reliable replication factor for the test to run a load on
+        :return:
+        """
+        n_db_nodes = self.params.get('n_db_nodes')
+        if isinstance(n_db_nodes, int):  # legacy type
+            return n_db_nodes
+        elif isinstance(n_db_nodes, str):  # latest type to support multiple datacenters
+            return min([int(nodes_num) for nodes_num in n_db_nodes.split() if nodes_num and int(nodes_num) > 0])
```

Comment:
1. Don't need to build a list for `min()`.
2. Result of `str.split()` doesn't contain empty strings.
3. Pretty sure that `int(nodes_num) > 0` can be replaced with `nodes_num != "0"`

Reply:
1. Agree, but i think it will be less readable, also you will have to deal with negatives inside lambda, which won't make it nice.
2. Fixed
3. Technically there could be negatives